src/net/sourceforge/kolmafia/KoLConstants.java

```diff
@@ -73,7 +73,6 @@
 
 	String VERSION_NAME = "KoLmafia v20.7";
```

Comment:
If we want to enforce that all accesses are done through StaticEntity.java's getVersion(), getRevision(), VERSION_NAME / REVISION (at the very least) should be private fields that live in StaticEntity.